workflow-beta3/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+name: workflow-beta3
+home: https://github.com/deis/workflow
+version: 2.0.0-beta3
+description: For testing only!
+maintainers:
+- Deis Team <engineering@deis.com>
+details: |-
+  WARNING: this chart is for testing only! Features may not work, and there are likely to be bugs.
```

Comment:
Should we note that workflow-beta3 requires [`helm` 0.6.0](https://github.com/helm/helm/releases/tag/0.6.0) or later?